curvefs/conf/client.conf

```diff
@@ -245,9 +245,10 @@ diskCache.trimCheckIntervalSec=5
 # the interval of check to trim load file to s3
 diskCache.asyncLoadPeriodMs=5
 # start trim file when disk cache use ratio is Greater than fullRatio,
-# util less than safeRatio
+# util less than safeRatio * trimeRatio / 100
```

Comment:
trimeRatio -> trimRatio
Is the time point of start trim reaching fullRatio or trimRatio? Fix code comments if reach trimRatio

Reply:
> trimeRatio -> trimRatio Is the time point of start trim reaching fullRatio or trimRatio? Fix code comments if reach trimRatio

fullRatio start to trim